chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -387,6 +387,52 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
             dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: msg?.payload?.minimized });
         });
 
+
+
+        BroadcastService.getMessageByEventName("sendCustomEvent").subscribe((event: object) => {
+            let payload = {};
+            if ("payload" in event) payload = event.payload as object;
+            console.log("debugging: sendCustomEvent is received", event);
+            if ("customEventName" in payload && payload.customEventName && typeof payload.customEventName === "string" 
+                && "customEventValue" in payload && payload.customEventValue
+            ) {
+                try {
+                    const customEventValueStr = typeof payload.customEventValue === "string" ? payload.customEventValue : JSON.stringify(payload.customEventValue);
```

Comment:
shall we add a util function to validate when an object is string?, is repeated in this logic 3 times